docs/advanced/api_examples.yml

```diff
@@ -281,6 +281,29 @@
 
 -
   id:
+    "create container with environment"
+
+  doc: |
+    Create a new container.
```

Comment:
This isn't unique. And I think this prevents it being shown in the docs:
 * http://build.clusterhq.com/results/docs/create-container-with-environment-FLOC-1544/advanced/api.html